Source/SpatialGDK/Public/SpatialInterop.h

```diff
@@ -10,6 +10,14 @@
 #include "SpatialUnrealObjectRef.h"
 #include "SpatialInterop.generated.h"
 
+namespace improbable
+{
+	namespace unreal
+	{
+		class GlobalStateManagerData;
```

Comment:
How come this is back here?

Reply:
See @danielimprobable's comment - basically due to build times, as including a (not strictly necessary) header can impact them. I also think it's cleaner design-wise to only include headers which are very tightly coupled with a class in the header.